src/aig-common/modules/standard/components/social-action/social-action-form/social-action-form.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
this component not follow the standard indication of NewUpdateForm.
Fix it

Reply:
This component also was modified as *-new-update-form component.